webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -81,12 +87,22 @@ export const ErrataTab = () => {
     </DropdownItem>,
   ];
 
+  const applyByKatelloAgent = () => {
+    const selected = fetchBulkIds();
+    if (!isEmpty(selected)) {
+      const parameters = { bulk_errata_ids: JSON.stringify(selected) };
+      setIsBulkActionOpen(false);
+      clearSelections();
+      dispatch(applyViaKatelloAgent(hostId, parameters));
+    }
+  };
+
   const actionButtons = (
     <Split hasGutter>
       <SplitItem>
         <ActionList isIconList>
           <ActionListItem>
-            <Button isDisabled> {__('Apply')} </Button>
+            <Button isDisabled={selectedCount === 0} onClick={applyByKatelloAgent}> {__('Apply')} </Button>
```

Comment:
This Apply button works via Katello agent, and the one on Traces works via REX.  We should probably think about how we can be more consistent there, but that's probably outside the scope here

Reply:
Agreed. I think once we update the Errata page with your Rex work, we can try to be more consistent.